manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml

```
@@ -11,18 +11,16 @@ resources:
 images:
   - name: docker.io/kubeflowkatib/katib-controller
     newName: docker.io/kubeflowkatib/katib-controller
-    newTag: latest
+    newTag: v0.17.0
```

```suggestion
    newTag: latest
```

The head branch should use the `latest` image.